src/module/implements/implement.js

```diff
@@ -39,20 +42,19 @@ class Implement {
   async createEffectsOnItem(item) {
     const implement = await fromUuid(item);
 
-    if (this.implementItem) this.deleteEffectsOnItem();
+    if (this.item) this.deleteEffectsOnItem();
 
     const implementRules = implement.system?.rules ?? [];
     for (const rule of this.rules) {
       implementRules.push(rule);
     }
 
     implement.update({ _id: implement._id, "system.rules": implementRules });
-    this.implementItem = implement.uuid;
+    this.itemId = implement.id;
```

Comment:
Should `this.item` also be updated to point to `implement` here?

Reply:
Yep, missed that one. Thanks!